cassandra-dac/src/test/java/org/sunbird/cassandra/CassandraOperationImplTest.java

```diff
@@ -0,0 +1,635 @@
+import java.text.MessageFormat;
+import java.util.*;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.CassandraUtil;
+import org.sunbird.common.Constants;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.PropertiesCache;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.CassandraConnectionManagerImpl;
+import org.sunbird.helper.CassandraConnectionMngrFactory;
+import org.sunbird.helper.ServiceFactory;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+/** @author kirti. Junit test cases */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  Cluster.class,
+  CassandraOperationImpl.class,
+  Uninterruptibles.class,
+  PreparedStatement.class,
+  BoundStatement.class,
+  Session.class,
+  Metadata.class,
+  CassandraConnectionMngrFactory.class,
+  ResultSet.class,
+  CassandraUtil.class,
+  Cluster.Builder.class,
+  Select.class,
+  Row.class,
+  ColumnDefinitions.class,
+  String.class,
+  Select.Where.class,
+  Select.Builder.class,
+  QueryBuilder.class,
+  Select.Selection.class,
+  Delete.Where.class,
+  Delete.Selection.class
+})
+@PowerMockIgnore("javax.management.*")
+public class CassandraOperationImplTest {
+
+  private static Cluster cluster;
+  private static Session session = PowerMockito.mock(Session.class);
+  private static PreparedStatement statement;
+  private static ResultSet resultSet;
+  private static Select selectQuery;
+  private static Select.Where where;
+  private static Delete.Where deleteWhere;
+  private static Select.Builder selectBuilder;
+  private static Metadata metadata;
+  private static CassandraOperation operation;
+  private static Map<String, Object> address = null;
+  private static Map<String, Object> dummyAddress = null;
+  private static PropertiesCache cach = PropertiesCache.getInstance();
+  private static String host = cach.getProperty("contactPoint");
+  private static String port = cach.getProperty("port");
+  private static String cassandraKeySpace = cach.getProperty("keyspace");
+  private static final Cluster.Builder builder = PowerMockito.mock(Cluster.Builder.class);
+  private static BoundStatement boundStatement;
+  private static Select.Selection selectSelection;
+  private static Delete.Selection deleteSelection;
+  private static Delete delete;
+  private static KeyspaceMetadata keyspaceMetadata;
```

Comment:
is every variable above referred in multiple methods??

Reply:
For example deleteWhere, builder are required only in initialization method.
Keyspace metadata is used only in specific test-method